Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/CRC.h

```diff
@@ -17,3 +17,5 @@ namespace ScriptCanvas
         Data::CRCType FromString(Data::StringType value);
     } // namespace CRCFunctions
 } // namespace ScriptCanvas
+
+#include <Include/ScriptCanvas/Libraries/Math/CRC.generated.h>
```

Comment:
Just curious, is there any reason this needs to be included in both the header and cpp file?

Reply:
Yeah, I am looking for an answer for this as well.

Adding includes to the headers cost a lot, especially since ScriptCanvas is the slower CMake targets to build.